libs/libGamedata/lua/GameDataLoader.cpp

```diff
@@ -94,7 +83,9 @@ void GameDataLoader::Include(const std::string& filepath)
             throw LuaIncludeError("File is outside the lua data directory!");
         const auto oldCurFile = curFile_;
         curFile_ = absFilePath;
-        errorInIncludeFile_ |= !loadScript(absFilePath);
+        ++curIncludeDepth_;
+        if(!loadScript(absFilePath))
+            throw std::runtime_error(helpers::format("Include file '%1%' cannot be included", filepath));
```

Comment:
why is this not a LuaIncludeError?

Reply:
Because I wanted a different error. `LuaIncludeError` here is more like `optional<string> = try_include()` But here there is no additional error message because `loadScript` or inside its Lua execution already does all reporting. So we just need to pass the error up.